apps/ui/vite.config.ts

```diff
@@ -1,7 +1,13 @@
 import react from '@vitejs/plugin-react';
+import path from 'path';
 import { defineConfig } from 'vite';
 
 // https://vitejs.dev/config/
 export default defineConfig({
 	plugins: [react()],
+	resolve: {
+		alias: {
+			'@assets': path.resolve(__dirname, './src/assets'),
```

Comment:
which means this one would need to change too
```suggestion
			'@/assets': path.resolve(__dirname, './src/assets'),
```